### PR TITLE
[CAPT-518] Add degree question to check your answers page

### PR DESCRIPTION
**Pull request by vassyz**
Screenshot 2022-07-27 at 11 23 28

### CODE REVIEW

File: app/models/current_claim.rb [hunk 1]
```diff
@@ -144,6 +144,12 @@ def policy_year
     end
   end
 
+  def eligible_eligibility
```
Comment (kenfodder) [inline app/models/current_claim.rb:147]: Wondering if this could just be `def eligibility`?

File: app/helpers/claims_helper.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module ClaimsHelper
-  def eligibility_answers(claim)
-    claim.policy::EligibilityAnswersPresenter.new(claim.eligibility).answers
+  def eligibility_answers(current_claim)
+    current_claim.policy::EligibilityAnswersPresenter.new(current_claim.eligible_eligibility).answers
```
Comment (kenfodder) [inline app/helpers/claims_helper.rb:3]: If it's just `def eligibility` then this can just stay the same.

**Review by kenfodder — APPROVED** [review]

File: app/models/concerns/early_career_payments_helpable.rb [hunk 3]
```diff
@@ -36,4 +36,12 @@ def no_entire_term_contract?
   def trainee_teacher?
     nqt_in_academic_year_after_itt == false
   end
+
+  def qualification_name
```
Comment (nigel-lowry) [inline app/models/concerns/early_career_payments_helpable.rb:40]: We don't need to worry about this now, and my module naming wasn't great here, but I planned to delete this entire module. Currently, it's where the methods are for the implementation of the ineligible screens. That whole lot is going to be replaced when I tackle the updated ineligible screens. We could keep this module around but it'll literally only have your new method in it.

Is there anywhere else your new method could go?

File: app/models/current_claim.rb (hunk 1 above)
Comment (nigel-lowry) [reply app/models/current_claim.rb:147]: @vassyz do you have to maintain an interface or something? Is that why it's got this name?

**Review by nigel-lowry — APPROVED** [review]

File: app/models/current_claim.rb (hunk 1 above)
Comment (vassyz) [reply app/models/current_claim.rb:147]: No, it's literally an eligible eligibility (if there is one).

File: app/models/concerns/early_career_payments_helpable.rb (hunk 3 above)
Comment (vassyz) [reply app/models/concerns/early_career_payments_helpable.rb:40]: When you guys had a go at this, you coped the method from the ECP to the LUP eligibility model: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/compare/master...capt-518-degree-question-response-in-teacher-journey#diff-d5a4e998dcde35819d9253160606c2aebbedcab062cdf7e8306e51172fdf142fR77. I just looked for a module that both eligibilities have access to so I don't have duplication. Happy to move it somewhere else. Or have the duplication if you think that works better.

Comment (nigel-lowry) [reply app/models/concerns/early_career_payments_helpable.rb:40]: Would `app/helpers/early_career_payments_helper.rb` be alright?

**Review by kenfodder — APPROVED** [review]

**Pull request merged**